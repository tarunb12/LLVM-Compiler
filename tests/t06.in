int main() {
    int max = 5
    int i
    for (i = 0; i <= max; i++) {
        if (i < max) printf("%d steps\n", i)
        else printf ("%dth and final step", i)
    }
    return 0
}