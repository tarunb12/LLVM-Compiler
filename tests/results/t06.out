0 steps
1 steps
2 steps
3 steps
4 steps
5th and final step